src/conponent/tracker/PlayerStatsTrackerSecondary.java

```diff
@@ -0,0 +1,90 @@
+package conponent.tracker;
+
+/**
+ * Implements all secondary methods by using kernal methods. Overrides object
+ * methods using kernal methods.
+ *
+ * @author Ryan Chou
+ */
+public abstract class PlayerStatsTrackerSecondary
```

Comment:
The enhanced interface is missing. I thought maybe it was the PlayerStatsTracker file, but now I'm thinking it's this file. It's unclear.